zerver/lib/streams.py

```diff
@@ -285,35 +285,36 @@ def access_stream_for_send_message(
         else:
             raise JsonableError(_("User not authorized for this query"))
 
-    if is_cross_realm_bot_email(sender.delivery_email):
-        return
+    if not stream.deactivated:
```

Comment:
Lets raise `JsonableError` for deactivated stream instead of indenting the entire block

Reply:
Done.